docs/reference_guide.md

```diff
@@ -2752,6 +2754,31 @@ Attaching 1 probe...
 ^C
 ```
 
+## 25. `path()`: Return full path
+
+Syntax:
+- `path(struct path *path)`
+
+Return full path referenced by struct path pointer in argument.
```

Comment:
One more thing: maybe it'd be good to document there's a whilelist of kernel functions you can use this on. Could be confusing for users who get load errors when they have a correct handle to `struct path`.

Reply:
right, I added a sentence about that.. I think we need some way to get the whitelist info from kernel, I'll try to add something